aws/resource_aws_cloudwatch_event_target.go

```diff
@@ -146,7 +146,7 @@ func resourceAwsCloudWatchEventTarget() *schema.Resource {
 							Type:         schema.TypeString,
 							Optional:     true,
 							Default:      events.LaunchTypeEc2,
-							ValidateFunc: validation.StringInSlice(events.LaunchType_Values(), false),
+							ValidateFunc: validation.StringInSlice(append(events.LaunchType_Values(), []string{""}...), false),
```

Comment:
Exactly the right place, but I would just consolidate this by using our `validation.Any` helper method 
```suggestion
							ValidateFunc: validation.Any(
								validation.StringIsEmpty,
								validation.StringInSlice(events.LaunchType_Values(), false),
							),
```